packages/drivers/odsp-driver/src/odspDocumentServiceFactoryWithCodeSplit.ts

```diff
@@ -7,14 +7,14 @@ import { IDocumentServiceFactory } from "@fluidframework/driver-definitions";
 import { IPersistedCache } from "./odspCache";
 import { OdspDocumentServiceFactoryCore } from "./odspDocumentServiceFactoryCore";
 import { HostStoragePolicy } from "./contracts";
-import { StorageTokenFetcher, PushTokenFetcher } from "./tokenFetch";
+import { OdspResourceTokenFetcher } from "./tokenFetch";
 
 export class OdspDocumentServiceFactoryWithCodeSplit
     extends OdspDocumentServiceFactoryCore
     implements IDocumentServiceFactory {
     constructor(
-        getStorageToken: StorageTokenFetcher,
-        getWebsocketToken: PushTokenFetcher,
+        getStorageToken: OdspResourceTokenFetcher,
+        getWebsocketToken: OdspResourceTokenFetcher,
```

Comment:
Probably keeping them separate is okay. I am not sure if signature would ever deviate in future. This conceptually looks more and more like generic TokenProvider signature but I think it would be better taken as separate change.